From 9af810af6c6d4a298138611f8168e8236b9e4520 Mon Sep 17 00:00:00 2001
From: Xiao Chen <xiao@apache.org>
Date: Tue, 14 Feb 2017 09:46:55 -0800
Subject: [PATCH 2211/2848] HDFS-11390. Add process name to httpfs process.
 Contributed by Weiwei Yang.

(cherry picked from commit 92dbdbc490a527054388dc2fc6751aa8268856ef)
(cherry picked from commit 8f9aae0193bc3696cb2047ff5f8d019579ad345f)

Change-Id: I449cfa4fa907d7358b2f1478d6562d2e0b4bab3e
---
 .../hadoop-hdfs-httpfs/src/main/sbin/httpfs.sh     |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs-httpfs/src/main/sbin/httpfs.sh b/hadoop-hdfs-project/hadoop-hdfs-httpfs/src/main/sbin/httpfs.sh
index ef0b7fc..8cc1431 100644
--- a/hadoop-hdfs-project/hadoop-hdfs-httpfs/src/main/sbin/httpfs.sh
+++ b/hadoop-hdfs-project/hadoop-hdfs-httpfs/src/main/sbin/httpfs.sh
@@ -84,7 +84,8 @@ source ${HADOOP_LIBEXEC_DIR:-${BASEDIR}/libexec}/httpfs-config.sh
 #
 print "Using   CATALINA_OPTS:       ${CATALINA_OPTS}"
 
-catalina_opts="-Dhttpfs.home.dir=${HTTPFS_HOME}";
+catalina_opts="-Dproc_httpfs";
+catalina_opts="${catalina_opts} -Dhttpfs.home.dir=${HTTPFS_HOME}";
 catalina_opts="${catalina_opts} -Dhttpfs.config.dir=${HTTPFS_CONFIG}";
 catalina_opts="${catalina_opts} -Dhttpfs.log.dir=${HTTPFS_LOG}";
 catalina_opts="${catalina_opts} -Dhttpfs.temp.dir=${HTTPFS_TEMP}";
-- 
1.7.9.5

